### ISSUE TITLE
[Term Entry] Python:NumPy Math Methods cos

### ISSUE THREAD

**Issue by codecademy-docs** [Codecademy/docs#4715]
### Reason/inspiration (optional)

"We would like a new term entry on the `Math Methods` term for [Numpy](https://www.codecademy.com/resources/docs/numpy): cos. The entry should go in a new file under `docs/content/numpy/concepts/math-methods/terms/cos/cos.md`.

The entry should describe the term and how it is used. Also include:
- An "Example" section that explains readers how to use the cos method on a given array.
- Should contain informations for functions and syntax.
- Should have a python codebyte with working code snippet showcasing this function.
Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

**Comment by Jfri3ss** [comment]
@codecademy-docs

I want to solve the problem. I am working through a CodeCademy course and contributing to CodeCademy Docs.

Please assign the issue to me.

**Comment by letsbuildanything** [comment]
@Jfri3ss You’re assigned :tada:  In addition to the documents linked in the description, please also look at the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide).  After creating a PR, the maintainer(s) (with the collaborator label) will add comments/suggestions to address any revisions prior to approval.
Is this your first contribution to Codecademy Docs? If so, we’re curious to know how you found out about contributing to Docs.

**Comment by Jfri3ss** [comment]
@letsbuildanything Yes, this is my first contribution. I am working on the iOS developer course on Codecademy.

One of the projects is to contribute to Codecademy Docs.

**Comment by Jfri3ss** [comment]
@mamtawardhani Hey man, I updated my Codecademy profile with my Github user name after I started the project. 

Can I get credit as the author on the CodeCademy website? Link below: 

https://www.codecademy.com/resources/docs/numpy/math-methods/cos

**Comment by ishg-153** [comment]
@Jfri3ss It takes a while for your GitHub username to show up as author once the username is added to profile. You check it again in a couple of days.

**Comment by mamtawardhani** [comment]
> @mamtawardhani Hey man, I updated my Codecademy profile with my Github user name after I started the project.
> 
> Can I get credit as the author on the CodeCademy website? Link below:
> 
> https://www.codecademy.com/resources/docs/numpy/math-methods/cos

Hey @Jfri3ss, this is because your codecademy account isn't updated with your GitHub username

To solve this you will have to go to your account at www.codecademy.com and edit your profile. Add your GitHub username there are try to log out and log in once with the GitHub account. This will be solved. 
Hope this helps! 😃
